test/components/Icon.test.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { shallow } from 'enzyme';
+
+import Icon from '../../src/Icon';
+
+describe('Icon', () => {
+
+  it('renders correctly', () => {
+    const component = renderer.create(
+      <Icon icon='foo' />
+    );
+    expect(component).toMatchSnapshot();
```

Comment:
my authoritative source for this is: https://twitter.com/VilleImmonen/status/772488155584983041